config/nrfconnect/BUILD.gn

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+
+import("//${chip_root}/gn/chip/tests.gni")
+
+assert(current_os == "zephyr")
+
+group("nrfconnect") {
+  deps = [ "${chip_root}/src/lib" ]
+}
+
+if (chip_build_tests) {
+  import("//${chip_root}/gn/chip/chip_test_group.gni")
+
+  group("check") {
+    deps = [ ":tests_run" ]
+  }
+
+  chip_test_group("tests") {
+    deps = [
+      "${chip_root}/src/ble/tests",
```

Comment:
Maybe we should move this list to

//src:tests ?

Reply:
I have removed this test group for the time being. I will apply your suggestion while working on the native posix test runner for zephyr.